include/query.php

```diff
@@ -100,16 +100,27 @@ public function get_queried_taxonomies() {
 	 * Optimized for (and requires) WP 3.5+.
 	 *
 	 * @since 2.2
+	 * @since 3.3 Accepts now an array of languages.
 	 *
-	 * @param PLL_Language $lang Language object.
+	 * @param PLL_Language|PLL_Language[] $langs Language object(s).
 	 * @return void
 	 */
-	public function set_language( $lang ) {
+	public function set_language( $langs ) {
+		if ( $langs instanceof PLL_Language ) {
+			$langs = array( $langs );
+		}
```

Comment:
For me, it's more logical to check it like this:
```suggestion
	public function set_language( $languages ) {
		if ( ! is_array( $languages ) ) {
			$languages = array( $languages );
		}
```